modules/reindex/src/main/java/org/elasticsearch/index/reindex/Reindexer.java

```diff
@@ -218,11 +217,6 @@ protected void finishHim(Exception failure, List<BulkItemResponse.Failure> index
             return super.buildScriptApplier();
         }
 
-        @Override
-        protected BulkRequest buildBulk(Iterable<? extends ScrollableHitSource.Hit> docs) {
```

Comment:
As we removed only overriding method we can change it back to `private` in parent class